docs/code-of-conduct-response.rst

```diff
@@ -0,0 +1,58 @@
+
+Write the Docs Code of Conduct - Reporting Guide
+================================================
+
+.. _report: conduct@writethedocs.org
+
+In general, in any Write the Docs space (conferences, meetup, Slack, forum), if you believe someone is violating the code of conduct we ask that you report it to the staff by emailing report_. All reports will be kept confidential. In some cases we may determine that a public statement will need to be made. If that's the case, the identities of all victims and reporters will remain confidential unless those individuals instruct us otherwise.
```

Comment:
If the report includes a minor, there is a chance that Mandatory Reporting laws would apply; therefore, we might want to clarify the "All reports will be kept confidential" to reflect our legal obligation.